Blaise-Shyaka/politico-react
# [#172014389] Create landing page

# What does this PR do?

It creates a home page allowing users to view a short description of the application and a sign in form.

# Description of the tasks to be completed

- Create a simple home page with a short description
- Add a sign in form
- Make the page responsive

# How should this be manually tested?

- Clone this repository
- `cd politico-react`
- In the terminal, run `npm start` to open the app in the browser
- Check the presence of a short description of the application
- Resize the browser to verify its responsiveness

# What are the relevant pivotal tracker stories?

#172014389

# Screenshots

![Screenshot (165)](https://user-images.githubusercontent.com/52489421/77815088-c8e90400-70bf-11ea-97b5-8e95756fd0cf.png)

![Screenshot (167)](https://user-images.githubusercontent.com/52489421/77815089-d2726c00-70bf-11ea-8db9-208c9c51e454.png)

![Screenshot (166)](https://user-images.githubusercontent.com/52489421/77815093-e322e200-70bf-11ea-9a6f-945460a383f0.png)
